xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-ui/src/main/resources/SkinsCode/XWikiSkinsSheet.xml

```diff
@@ -78,17 +78,34 @@
   #set ($deprecatedProperties = [])
   #set ($className = 'XWiki.XWikiSkins')
   #set ($class = $doc.getObject($className).xWikiClass)
+  {{html wiki='true'}}
+  &lt;dl&gt;
   #foreach ($prop in $class.properties)
     #if ($prop.classType != 'TextArea')
-    ; {{html}}&lt;label for="${className}_$escapetool.xml($doc.getObject($className).getNumber())_$escapetool.xml($prop.name)"&gt;$escapetool.xml($prop.prettyName)&lt;/label&gt;{{/html}}
-    : #if($prop.name == 'logo')## Avoid line break
-      {{attachmentSelector classname="${className}" property="${services.rendering.escape($prop.name, 'xwiki/2.1')}" filter="png,jpg,gif,svg" displayImage="true"/}}## Avoid line break
-    #else$doc.display($prop.name)#end
+    &lt;dt&gt;
+      &lt;label for="${className}_$escapetool.xml($doc.getObject($className).getNumber())_$escapetool.xml($prop.name)"&gt;
+        $escapetool.xml($prop.prettyName)
```

Comment:
As we're now in an HTML + wiki context, all those escaped variables here now also need escaping for XWiki syntax. There shouldn't be any security issues as `$escapetool.xml` already escapes `{` but weird things could still happen.